### PR TITLE
Fix bug: NullPointerException etc.

### PR DESCRIPTION
**Pull request by Jave-Chen**
## What is the purpose of the pull request
#1720 Fix bug: NullPointerException etc.

## Brief change log
1. NullPointerException 
-  Add null check
2. Use try-with-resources or close this "Scanner" in a "finally" clause.
- Add try-with-resources on FSDataInputStream 
3. Not enough arguments.
- Add argument with string.

### CODE REVIEW

File: dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/model/DateInterval.java [hunk 1]
```diff
@@ -35,13 +35,18 @@ public DateInterval(Date beginTime, Date endTime){
 
     @Override
     public boolean equals(Object obj) {
-        try{
-            DateInterval dateInterval = (DateInterval) obj;
-            return startTime.equals(dateInterval.getStartTime()) &&
-                    endTime.equals(dateInterval.getEndTime());
-        }catch (Exception e){
+        if (obj != null) {
```
Comment (Technoboy-) [inline dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/model/DateInterval.java]: why use try/catch surround equals method?

Comment (Jave-Chen) [reply dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/model/DateInterval.java]: check whether obj is null

**Comment by qiaozhanwei** [comment]
please resolve conflicts

**Comment by Jave-Chen** [comment]
> please resolve conflicts

done

**Pull request merged**